eval.py

```diff
@@ -56,7 +60,8 @@ async def run_linkup_policy(
     question: str,
     policy_args: dict[str, Any] | None,
 ) -> Tuple[str, None]:
-    """Run linkup policy in a thread to avoid blocking."""
+    """Run linkup policy in a thread to avoid blocking.
```

Comment:
I think this comment is not super accurate (you should specify the arguments you mention should be in the policy args) and the values are not right (typically you're missing the localhost port), no?
Anyway I'm not sure it's worh it having it in the docstring, this is a public repository for people to evaluate linkup on simple-qa, so maybe let's not bother mentioning internal development stuff in there, it will make this more confusing, wdyt? 🙂